cfme/intelligence/reports/widgets/report_widgets.py

```diff
@@ -42,26 +43,30 @@ class EditReportWidgetView(BaseEditDashboardWidgetView, ReportWidgetFormCommon):
     pass
 
 
+@attr.s
 class ReportWidget(BaseDashboardReportWidget):
 
     TYPE = "Reports"
     TITLE = "Report"
     pretty_attrs = ["description", "filter", "visibility"]
-
-    def __init__(self, title, description=None, active=None, filter=None, columns=None, rows=None,
-            timer=None, visibility=None):
-        self.title = title
-        self.description = description
-        self.active = active
-        self.filter, self.subfilter, self.repfilter = filter
+    filter = attr.ib(default=None)
+    subfilter = attr.ib(init=False)
+    repfilter = attr.ib(init=False)
+    rows = attr.ib(default=None)
+    timer = attr.ib(default=None)
```

Comment:
**Question**

Default for timer here is None, dict above for ChartWidget?

Reply:
My bad, fixed